src/timesignature.ts

```diff
@@ -92,15 +92,15 @@ export class TimeSignature extends StaveModifier {
       assertIsValidFraction(timeSpec);
     }
 
-    const [topDigits, botDigits] = timeSpec.split('/').map((number) => number.split(''));
+    const parts = timeSpec.split('/');
 
     return {
       num: true,
-      glyph: this.makeTimeSignatureGlyph(topDigits, botDigits),
+      glyph: this.makeTimeSignatureGlyph(parts[0] ?? '', parts[1] ?? ''),
```

Comment:
The `?? ''` look like silent failures here. Shouldn't the assertion above throw if the parts are going to be empty?

Reply:
Now it is possible to have only one part in the case of '+'

Reply:
See my comment about the regex test above. (If I understand your comment correctly, you're saying this accepts `4+` when it should reject it, right?)

Reply:
No, I am syaing that this accepts `+` this is done as described below to avoid parsing complex signatures.